engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/typed/AvroSchemaTypeDefinitionExtractor.scala

```diff
@@ -51,5 +53,18 @@ object AvroSchemaTypeDefinitionExtractor {
         Typed.empty
     }
   }
+}
+
+object AvroSchemaTypeDefinitionExtractor {
 
+  private lazy val withoutOptionallyFieldsExtractor = new AvroSchemaTypeDefinitionExtractor(skippNullableFields = true)
+
+  private lazy val withOptionallyFieldsExtractor = new AvroSchemaTypeDefinitionExtractor(skippNullableFields = false)
+
+  def typeDefinition(schema: Schema, skippNullableFields: Boolean = false): TypingResult = {
```

Comment:
let's not use default boolean parameters, they can be **really** misleading :)
Let's add method: typeDefinitionWithoutNullables to be clear?

Reply:
Ok.